### DISCUSSION TITLE
Scott Pilgrim/Ubisoft Connect stuttering

### DISCUSSION THREAD

**Discussion by NeoRootPain** [lutris/lutris#4243] [Q&A]
I've downloaded Lutris in order to play Scott Pilgrim Vs. The World: The Game on Ubisoft Connect, and surprisingly, things run pretty well all things considered, but although the performance itself is good, there's a lot of random stutters

I've tried using different proton versions and even running it with the virtual desktop option, but nothing really helps
Is there any sort of launch options I need to use or even use a custom wine version in order to run it without the stuttering?

**Comment by strycore** [comment]
By default, Lutris uses what will give you the best performance. The virtual desktop helps with windowing or resolution issues but not with performance. 

You can go to Preferences > Hardware Information and paste the output here to see if your setup should be able to run the game.

**Reply by NeoRootPain** [reply]
The game used to run fine on Windows, but for some reason Lutris is having trouble running it

Here's the output
[System]
OS:              Artix Linux rolling n/a
Arch:            x86_64
Kernel:          5.17.4-artix1-1
Desktop:         XFCE
Display Server:  x11

[CPU]
Vendor:          AuthenticAMD
Model:           AMD FX(tm)-8150 Eight-Core Processor
Physical cores:  4
Logical cores:   8

[Memory]
RAM:             15.6 GB
Swap:            0.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce GT 1030/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 510.60.02
OpenGL Core:     4.6.0 NVIDIA 510.60.02
OpenGL ES:       OpenGL ES 3.2 NVIDIA 510.60.02
Vulkan:          Supported

**Reply by strycore** [reply]
Do you have gamemode installed? That usually helps with stuttering. 
There's also going to be some stuttering when the shaders initially compile and there's no way around that except playing the game for a while and get the shaders cached.

**Reply by NeoRootPain** [reply]
> Do you have gamemode installed? That usually helps with stuttering. There's also going to be some stuttering when the shaders initially compile and there's no way around that except playing the game for a while and get the shaders cached.

Installing Feral Gamemode actually solved the issue
Thanks a lot for the help, now it runs perfectly!
